cpp/test/CMakeLists.txt

```diff
@@ -255,6 +255,7 @@ if(BUILD_TESTS)
     NAME
     NEIGHBORS_TEST
     PATH
+    test/neighbors/ann_cagra/test_float_uint32_t.cu
```

Comment:
do we need the other 3 files in `test/ann_cagra/` - if we're only compiling this one specialization?